### PR TITLE
Added drone file

### PR DESCRIPTION
**Pull request by nabim777**
This PR adds the drone config file with the following pipelines:
- Check Starlark
- unit test

### CODE REVIEW

File: .drone.star [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+            ],
+        },
+    }]
+
+def installPnpm():
+    return [{
+        "name": "pnpm-install",
+        "image": OC_CI_NODEJS,
+        "commands": [
+            'npm install --silent --global --force "$(jq -r ".packageManager" < package.json)"',
```
Comment (saw-jan) [inline .drone.star:70]: There is no `packageManager` in the `package.json`.
Please, add this line
```json
"packageManager": "pnpm@8.14.1",
```

File: .drone.star [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+        "kind": "pipeline",
+        "type": "docker",
+        "name": "unit-tests",
+        "steps": installPnpm() +
+                 [
+                     {
+                         "name": "unit-tests",
+                         "image": OC_CI_NODEJS,
+                         "commands": [
+                             "pnpm run coverage",
```
Comment (SagarGi) [inline .drone.star:53]: The pipe line says running unit tests but why there is only command for running coverage only? or the it runs unit tests?

**Review by SagarGi — CHANGES_REQUESTED** [review]

File: .drone.star (hunk 2 above)
Comment (saw-jan) [reply .drone.star:53]: `pnpm run coverage` runs the unit and coverage

File: pnpm-lock.yaml [hunk 3]
```diff
@@ -8,6 +8,12 @@ overrides:
   mime: ^3.0.0
 
 dependencies:
+  '@casl/ability':
+    specifier: ^6.3.3
+    version: 6.5.0
+  '@casl/vue':
+    specifier: ^2.2.1
+    version: 2.2.1(@casl/ability@6.5.0)(vue@3.3.4)
```
Comment (saw-jan) [inline pnpm-lock.yaml:16]: Some dependencies (in package.json) are not in the lock file entries. so we need this updated lock file. CI wants to install the dependencies using lock-file which is a good practice.

**Review by saw-jan — APPROVED** [review]
LGTM

**Review by SagarGi — APPROVED** [review]
LGTM :+1:

**Pull request merged**